worldedit-core/src/main/resources/com/sk89q/worldedit/world/registry/blocks.json

```diff
@@ -1700,7 +1700,7 @@
       "slipperiness": 0.6,
       "renderedAsNormalBlock": false,
       "liquid": false,
-      "solid": true,
+      "solid": false,
```

Comment:
Yes, it hasn't changed. I've generated the file myself and this was still true. How did you generate it? You may have used the wrong method depending on your mappings.